### PR TITLE
Add Vintage Story

### PR DESCRIPTION
**Pull request by jascrain**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. https://www.vintagestory.at/forums/topic/4018-permission-for-flatpak-package/
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge). (Domain is owned by Vintage Story developers)
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by jascrain** [comment]
cc @tyronx, if you want to check on the status of the Flathub package

**Comment by LeandroStanger** [comment]
bot, build at.vintagestory.VintageStory

You can trigger the builds as well in this PR

File: at.vintagestory.VintageStory.desktop [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```
Comment (bilelmoussaoui) [inline at.vintagestory.VintageStory.desktop:1]: Should be contributed upstream if possible, along with the metainfo file & icon

File: at.vintagestory.VintageStory.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            commands:
+                # cert-sync is needed to update mono's certificates, or
+                # communication with Vintage Story's auth server may fail
+                - /app/bin/mono ${MONO_PATH}/cert-sync.exe --quiet --user /etc/ssl/certs/ca-certificates.crt
+                - exec /app/bin/mono /app/extra/vintagestory/Vintagestory.exe
+          - type: file
+            path: at.vintagestory.VintageStory.desktop
+          - type: file
+            # from converting Vintage Story's assets/gameicon.xpm to png
+            path: at.vintagestory.VintageStory.png
```
Comment (bilelmoussaoui) [inline at.vintagestory.VintageStory.yaml:45]: The image says it's empty, is that expected?

Comment (jascrain) [reply at.vintagestory.VintageStory.yaml:45]: I don't know why the github description says it is empty. When I click on the dots > View file, the image shows.

File: at.vintagestory.VintageStory.desktop (hunk 1 above)
Comment (jascrain) [reply at.vintagestory.VintageStory.desktop:1]: Upstream already has a similar desktop file and icon. The issue is that they are in the extra-data file, so are not available during build time.

That, and their icon is in xpm format which I think is not supported.

**Comment by barthalion** [comment]
I don't have an account to test it so I trust @jascrain it works.

/merge

@jascrain, if someone from upstream team would like to have write access to the Flathub repo, please let us know.

**Comment by jascrain** [comment]
> @jascrain, if someone from upstream team would like to have write access to the Flathub repo, please let us know.

Possibly @tyronx would

**Pull request closed without merging**